zerver/views/documentation.py

```diff
@@ -86,26 +86,37 @@ def get_path(self, article: str) -> Tuple[str, int]:
         path = self.path_template % (article,)
         try:
             loader.get_template(path)
-            return (path, http_status)
+            return ArticlePath(
+                article=path, article_http_status=http_status, endpoint_name="", endpoint_method=""
+            )
         except loader.TemplateDoesNotExist:
-            return (self.path_template % ("missing",), 404)
+            return ArticlePath(
+                article=self.path_template % ("missing",),
+                article_http_status=404,
+                endpoint_name="",
+                endpoint_method="",
+            )
 
     def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         article = kwargs["article"]
         context: Dict[str, Any] = super().get_context_data()
-        (context["article"], http_status_ignored) = self.get_path(article)
+
+        Article_path = self.get_path(article)
```

Comment:
This should be `article_path = `; we never use capital letters in local variable names unless the name is an ALL_CAPS constant/settings.

Reply:
Changed the naming.